src/ResourceManager/PolicyInsights/Commands.PolicyInsights/Commands.PolicyInsights.csproj

```diff
@@ -51,18 +51,39 @@
     <Compile Include="Cmdlets\GetAzureRmPolicyEvent.cs" />
     <Compile Include="Cmdlets\GetAzureRmPolicyState.cs" />
     <Compile Include="Cmdlets\GetAzureRmPolicyStateSummary.cs" />
+    <Compile Include="Cmdlets\Remediation\StartAzureRmPolicyRemediation.cs" />
+    <Compile Include="Cmdlets\Remediation\StopAzureRmPolicyRemediation.cs" />
+    <Compile Include="Cmdlets\Remediation\RemoveAzureRmPolicyRemediation.cs" />
+    <Compile Include="Cmdlets\Remediation\GetAzureRmPolicyRemediation.cs" />
     <Compile Include="Common\ParameterHelpMessages.cs" />
+    <Compile Include="Common\PaginationHelper.cs" />
+    <Compile Include="Common\RemediationCmdletBase.cs" />
+    <Compile Include="Common\ResourceIdHelper.cs" />
     <Compile Include="Common\ParameterSetNames.cs" />
     <Compile Include="Common\PolicyInsightsCmdletBase.cs" />
     <Compile Include="Models\PolicyAssignmentSummary.cs" />
     <Compile Include="Models\PolicyDefinitionSummary.cs" />
+    <Compile Include="Models\Remediation\PSTypedErrorInfo.cs" />
+    <Compile Include="Models\Remediation\PSRemediationDeployment.cs" />
+    <Compile Include="Models\Remediation\PSRemediationDeplyomentSummary.cs" />
+    <Compile Include="Models\Remediation\PSErrorDefinition.cs" />
+    <Compile Include="Models\Remediation\PSRemediationFilter.cs" />
+    <Compile Include="Models\Remediation\PSRemediation.cs" />
     <Compile Include="Models\PolicyEvent.cs" />
     <Compile Include="Models\PolicyState.cs" />
     <Compile Include="Models\PolicyStateSummary.cs" />
     <Compile Include="Models\SummaryResults.cs" />
     <Compile Include="Properties\AssemblyInfo.cs" />
+    <Compile Include="Properties\Resources.Designer.cs">
+      <AutoGen>True</AutoGen>
+      <DesignTime>True</DesignTime>
+      <DependentUpon>Resources.resx</DependentUpon>
+    </Compile>
   </ItemGroup>
   <ItemGroup>
+    <None Include="Az.PolicyInsights.psd1">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </None>
```

Comment:
Don't add this file to this project.

Reply:
Removed.  

It was helpful having it in the csproj since it is very easy to forget to update it when adding new cmdlets.

Reply:
Understood. That csproj will be deleted at the end of next month anyway. But, it could cause a lot of issues for the final AzureRM build to have an Az psd1 file.